### DISCUSSION TITLE
Thank You, Maxnut.

### DISCUSSION THREAD

**Discussion by SeaBlockWho10** [maxnut/GDmenu#279] [General]
Thank you for your service for all of us with MegaOverlay, but I have now bought MegaHack V8 for $7.99. Thank you for giving us a free MegaHack version until we buy MegaHack! - SeaBlockWho10

**Comment by MysticAx0lotl** [comment]
Not everyone who uses GDMO can't afford MH, some of us (like myself) just prefer it.

**Reply by SeaBlockWho10** [reply]
I know..
